### PR TITLE
Question 39

### PR DESCRIPTION
**Pull request by rahulrana701**
I have added question 39 based on Destructuring Object Literal

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1514,3 +1514,121 @@ Here is an example of how to use the option C syntax:{import { default as myAlia
...
+
+- A: John Doe , 30 , New York 
+- B: John     , 30 ,  New York 
+- C: New York , 30 , John Doe
+- D: None of the above
+
+<details><summary><b>Answer</b></summary>
+<p>
+ 
+#### Answer:  B
```
Comment (kirandash) [inline README.md:1620]: Answer will be "A" not "B"

**Review by kirandash — CHANGES_REQUESTED** [review]
also please fix the merge conflict

**Comment by rahulrana701** [comment]
@kirandash I will create  a new pr for it , I somehow messed up this one, my question-39 is not visible in my repositery

**Comment by kirandash** [comment]
> @kirandash I will create a new pr for it , I somehow messed up this one, my question-39 is not visible in my repositery

ok. Go ahead. Or if you need help, feel free to let me know: I can do the changes on my side and push

**Pull request closed without merging**